src/rdkafka_admin.c

```diff
@@ -4331,7 +4397,7 @@ rd_kafka_DescribeAclsResponse_parse(rd_kafka_op_t *rko_req,
                      rd_kafka_AclBinding_free);
 
         for (i = 0; i < (int)res_cnt; i++) {
-                int8_t res_type;
+                rd_kafka_ResourceType_t res_type = RD_KAFKA_RESOURCE_UNKNOWN;
```

Comment:
This needs to be an int8_t since ..buf_read_i8() just performs a 1-byte memory copy, and we can't really know the size or layout of the enum.